expan/cli/cli.py

```diff
@@ -20,12 +20,28 @@ def __str__(self):
 
 
 def parse_metadata(filename):
+        """
+            Parse metadata file
+            Args:
+                filename - metadata file path
+        """
 	metadata_file = open(filename)
```

Comment:
there seems to be a tabs/spaces mixture here that python3 isn't happy about... https://travis-ci.org/zalando/expan/jobs/167632638